src/components/ScrollingList.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    ref="sliderContainer"
+    :class="[
+      'relative',
+      'flex',
+      'overflow-x-auto laptop:overflow-x-hidden',
+      'hover-animate',
+
+      'select-none',
+      'scrollbar-custom',
+    ]"
+    :style="{
+      '--animation-duration': durationTime,
+    }"
+    @mousedown="startDrag"
+    @mousemove="onDrag"
+    @mouseup="endDrag"
+    @mouseleave="endDrag"
+    @touchstart="startDrag"
+    @touchmove="onDrag"
+    @touchend="endDrag"
+  >
+    <ul class="flex animate-scroll">
+      <slot name="item" />
+    </ul>
+    <ul class="hidden laptop:flex animate-scroll delay">
+      <slot name="item" />
+    </ul>
+  </div>
+</template>
+<script>
+export default {
+  name: 'ScrollingList',
+  props: {
+    animationDuration: {
+      type: String,
+      default: '60s',
```

Comment:
Should we add unit here or just number?

Reply:
[🎨 Adjust animationDuration props type](https://github.com/likecoin/liker-land/pull/2002/commits/de1206246952f696f87aeb17c3af2991d2504bd5)